src/main/java/uk/gov/hmcts/juror/api/moj/domain/trial/Judge.java

```diff
@@ -42,16 +42,12 @@ public class Judge implements Serializable {
     @Length(max = 4)
     private String code;
 
-    @Column(name = "name")
-    @Length(max = 30)
-    private String name;
-
     @Column(name = "is_active")
     private boolean isActive;
 
     @Column(name = "description")
     @Length(max = 30)
-    private String description;
```

Comment:
why do field and database columns have different names? slightly counterintuitive but will work nonetheless